burn-tensor/src/tensor/backend/base.rs

```diff
@@ -111,18 +111,18 @@ pub trait ADBackend: Backend {
     ///
     /// # Arguments
     ///
-    /// * `tensor` - The tensor to compute the gradients for.
+    /// * `tensor` - The tensor is the last node of computational graph where the gradients are recomputed.
```

Comment:
`recomputed` => `computed` the gradients are not computed before `backward` is called.

Reply:
Fixed!